core/templates/core/includes/articles_list.html

```diff
@@ -14,12 +14,12 @@
         {% else %}
             <div class="col-12">
         {% endif %}
-            <h2>
```

Comment:
I changed the font from h3 to h2 because the article titles felt too big on the staff page.